runtime/src/main/java/org/corfudb/runtime/CorfuRuntime.java

```diff
@@ -939,6 +919,53 @@ private void pruneRemovedRouters(@Nonnull Layout layout) {
         });
     }
 
+    /**
+     * Helper function to fetch the latest layout from provided list of {@param layoutServers}
+     *
+     * @param layoutServers list layout servers
+     * @return Completed Future with the latest layout or an exception if no layout was found.
+     */
+    private CompletableFuture<Layout> getLatestLayout(List<String> layoutServers) {
+        CompletableFuture<Layout> latestLayoutFuture = null;
```

Comment:
Just make getLatestLayout return a Layout, if its null then return a dummy layout with -1 epoch. Then in fetchLayout you can wrap this with a future and return.